apps/server/src/auth/strategy/google-strategy.ts

```diff
@@ -0,0 +1,32 @@
+import { PassportStrategy } from '@nestjs/passport';
+import { Strategy, VerifyCallback } from 'passport-google-oauth2';
+import { Injectable } from '@nestjs/common';
+
+@Injectable()
+export class GoogleStrategy extends PassportStrategy(Strategy, 'google') {
+  constructor() {
+    super({
+      clientID: process.env.GOOGLE_CLIENT_ID,
+      clientSecret: process.env.GOOGLE_CLIENT_SECRET,
+      callbackURL: process.env.GOOGLE_CALLBACK_URL,
+      scope: ['email', 'profile'],
+    });
+  }
```

Comment:
Let's utilize `ConfigModule` to pull out the env variables instead of `process.env`

Reply:
updated the usage to `ConfigModule` to `process.env`